site/content/en/docs/reference/metrics.md

```diff
@@ -22,13 +22,15 @@ Use the following metrics to monitor the health of the kueue controllers:
 
 Use the following metrics to monitor the status of your ClusterQueues:
 
-| Metric name | Type | Description | Labels |
-| ----------- | ---- | ----------- | ------ |
-| `kueue_pending_workloads` | Gauge | The number of pending workloads. | `cluster_queue`: the name of the ClusterQueue<br> `status`: possible values are `active` or `inadmissible` |
-| `kueue_admitted_workloads_total` | Counter | The total number of admitted workloads. | `cluster_queue`: the name of the ClusterQueue |
-| `kueue_admission_wait_time_seconds` | Histogram | The time between a Workload was created until it was admitted. | `cluster_queue`: the name of the ClusterQueue |
+| Metric name | Type | Description                                                                    | Labels |
```

Comment:
Could you revert the formatting changes which don't result in the visual display of the table?
It is not immediate if these are just formatting or content changes.

Reply:
this seems done, feel free to mark as resolved